units/en/unit2/smolagents/why_use_smolagents.mdx

```diff
@@ -0,0 +1,62 @@
+![smolagents banner](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/smolagents/license_to_call.png)
+# Why use smolagents
+
+In this module, we will explore the pros and cons of using [smolagents](https://huggingface.co/blog/smolagents), helping you make an informed decision about whether it's the right framework for your needs.
```

Comment:
```suggestion
In this module, we will explore the pros and cons of using [smolagents](https://huggingface.co/docs/smolagents/en/index), helping you make an informed decision about whether it's the right framework for your needs.
```

Reply:
As you prefer on this one but I was thinking maybe it's better to put the documentation link but feel free to refuse this suggestion.